ci/travis/format.sh

```diff
@@ -254,6 +280,12 @@ format_changed() {
         fi
     fi
 
+    if command -v java >/dev/null; then
+       if ! git diff --diff-filter=ACRM --quiet --exit-code "$MERGEBASE" -- '*.java' &>/dev/null; then
+            git diff --name-only --diff-filter=ACRM "$MERGEBASE" -- '*.java' | format_java_files
```

Comment:
Missing `xargs -P 5`

Reply:
Since we format all files in one time. It's not necessay to add `-P 5`. `google-java-foramt.jar` will take care of it.

Reply:
You are right. No speed optimization with `-P 5`.